vividus-util/src/test/java/org/vividus/util/json/JsonUtilsTests.java

```diff
@@ -160,6 +161,14 @@ void shouldReadJson()
             ).toPrettyString());
     }
 
+    @Test
+    void shouldCreatePattern()
```

Comment:
Is the logic covered with this test obsolete now? 

```java
    @Test
    void shouldThrowExceptionInCaseOfNullPattern()
    {
        JsonParser jsonParser = mock(JsonParser.class);
        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class,
            () -> new PatternDeserializer().deserialize(jsonParser, null).toString());
        assertEquals("Pattern could not be empty", exception.getMessage());
    }
```

Reply:
I've checked this case during implementation, it can't happen anymore